builder/Dockerfile

```diff
@@ -1,17 +1,33 @@
-FROM golang AS builder
-RUN mkdir /app
-ADD . /app
-WORKDIR /app
-RUN CGO_ENABLED=0 GOOS=linux go build github.com/Azure/aks-periscope/cmd/aks-periscope
+# Builder
+FROM golang:alpine AS builder
 
+ENV GO111MODULE=on \
+    CGO_ENABLED=0 \
+    GOOS=linux \
+    GOARCH=amd64
+
+WORKDIR /build
+
+COPY go.mod .
+COPY go.sum .
+RUN go mod download
+
+COPY . .
```

Comment:
💡 **quick question**: Just checking for this copy the source and destination is same?